manifests/c/Cloudflare/Warp/2025.1.861.0/Cloudflare.Warp.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.4.3 $debug=NVS0.CRLF.5-1-26100-2161.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Cloudflare.Warp
+PackageVersion: 2025.1.861.0
+InstallerLocale: en-US
+InstallerType: msi
+Scope: machine
+InstallerSwitches:
+  Custom: /NORESTART
+UpgradeBehavior: install
+ReleaseDate: 2025-02-19
+Installers:
+- Architecture: x64
+  InstallerUrl: https://downloads.cloudflareclient.com/v1/download/windows/version/2025.1.861.0
+  InstallerSha256: 5D3D2FCCA081DE37A4CCFD8B29D7420CEF852B6171973C4F047C96731E1DD566
```

Comment:
>Use New Download URL

Same installer of https://github.com/microsoft/winget-pkgs/blob/66d215b8f4ae56655193bdb4b1c01c6f456a2ca2/manifests/c/Cloudflare/Warp/25.1.861.0/Cloudflare.Warp.installer.yaml#L28-L29?

Reply:
Reverted in https://github.com/microsoft/winget-pkgs/pull/236604

I hope this package will be a candidate in https://github.com/microsoft/winget-pkgs/issues/29323. 🙏